hack/gofmt.sh

```diff
@@ -10,14 +10,7 @@ if [ "${IS_CONTAINER}" != "false" ]; then
   mkdir /tmp/unit
   cp -r ./* /tmp/unit
   cd /tmp/unit
-  make fmt > /tmp/fmt-output.log
-  FILE_LENGTH="$(wc -l /tmp/fmt-output.log | awk '{ print $1 }')"
-  if [ "${FILE_LENGTH}" != "1" ]; then
-    echo "Formatting error! Please run 'make fmt' to correct the problem."
-    echo "The problematic files are listed below, after the command that should be run"
-    cat /tmp/fmt-output.log
-    exit 1
-  fi
```

Comment:
you should not remove this, it detects that there has been a change done by fmt. Since you run in a container, just calling make fmt will have no effect on the actual files (check the --volume parameter, set to ro)

Reply:
Same, it is taken care of already by #222